File: 은지/LV1/모의고사.js
function solution(answers) {
  var answer = []
  const N = answers.length
  // A, B, C의 답안
  var A = []
  for (let i = 0; i < N; i++) {
    A.push(i%5+1)
  }
  var B = []
  const b = [1, 3, 4, 5]
  for (let i = 0; i < N/2+1; i++) {
    B.push(2)
    B.push(b[i%4])
  }
  var C = []
  const c = [3, 1, 2, 4, 5]
  for (let i = 0; i < N/2+1; i++) {
    C.push(c[i%5])
    C.push(c[i%5])
  }
  // 답 체크
  var Check = [0, 0, 0]
  for (let i = 0; i < N; i++) {
    if (A[i] === answers[i]) {
      Check[0] += 1
    }
    if (B[i] === answers[i]) {
      Check[1] += 1
    }
    if (C[i] === answers[i]) {
      Check[2] += 1
    }
  }
  // 가장 높은 점수
  const maxV = Math.max(...Check)
  // 가장 높은 점수 학생 탐색
  for (let i=0; i<3; i++) {
    if (Check[i] === maxV) {
      answer.push(i+1)
    }
  }  
  return answer;
}

File: 은지/LV1/모의고사.py
def solution(answers):
    answer = []
    # 문제의 개수
    N = len(answers)
    # A의 답안
    A = [x%5+1 for x in range(N)]
    # B의 답안
    B = []
    b = [1, 3, 4, 5]
    for i in range(N//2+1):
        B.append(2)
        B.append(b[i%4])
    # C의 답안
    C = []
    c = [3, 1, 2, 4, 5]
    for i in range(N//2+1):
        C.append(c[i%5])
        C.append(c[i%5])
    # 답 체크
    Check = [0, 0, 0]
    for i in range(N):
        if A[i] == answers[i]:
            Check[0] += 1
        if B[i] == answers[i]:
            Check[1] += 1
        if C[i] == answers[i]:
            Check[2] += 1
    # 가장 높은 점수
    maxV = max(Check)
    # 가장 높은 점수 학생 탐색
    for i in range(3):
        if Check[i] == maxV:
            answer.append(i+1)
    return answer